src/components/forms/fragments/UnRegisteredBoatDetails.tsx

```diff
@@ -14,16 +14,16 @@ interface Props {
   fieldsNotRequired?: boolean;
   boatTypes?: BoatTypes;
   showBoatStorageType?: boolean;
+  boatStorageType?: boolean;
```

Comment:
a bit misleading: the name has word "type", but the value is boolean. maybe change the name to a more descriptive one?

Reply:
i name it after ticket description, it should be `boatHasTrailer` ?

Reply:
`boatIncludeTrailer` ?

Reply:
`boatStoredOnTrailer` would be the most accurate 🙂

Reply:
would be dev friendly for the future devs to rename the const in utils too

Reply:
Done!